crd/apis/network/v1/annotations.go

```diff
@@ -64,6 +66,20 @@ type InterfaceRef struct {
 	Interface *string `json:"interface,omitempty"`
 }
 
+// NICInfoAnnotation is the value of the nic-info annotation
+type NICInfoAnnotation []NICInfoRef
```

Comment:
see other types in this file, add proper kubebuilder flags.

Reply:
Done